js/controller.js

```diff
@@ -520,6 +544,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onPause: function(event, source, pauseReason) {
+     if(this.state.failoverInProgress){
```

Comment:
The spacing for this section is off.  Each one has an odd number of spaces.  Should be multiples of 2. Also need a space between ) and {